docs/config.json

```diff
@@ -766,6 +766,10 @@
               "title": "EC2 Tags",
               "slug": "/management/guides/ec2-tags/"
             },
+            {
+              "title": "GCP Tags and Labels",
```

Comment:
nit: this title is "tags and labels", but the page it links to is titled "labels and tags".

I would pick one and be consistent.